es6/assets.js

```diff
@@ -0,0 +1,10 @@
+/** @const {boolean} exists check whether assets constant is defined */
+const exists = !!window.NOS && !!window.NOS.ASSETS;
```

Comment:
Would love to see this exported as `assetsExist` or something!

Reply:
Doesnt seem so usefull in my eyes - it's also initialized if it doesn't exist, so it always exists

Reply:
True.